packages/devops/scripts/deployment/setupGoldMaster.sh

```diff
@@ -39,44 +39,44 @@ rm -rf lastpass-cli
 mkdir -p /home/ubuntu/.local/share/lpass
 
 # install puppeteer dependencies https://pptr.dev/15.3.0/troubleshooting#chrome-headless-doesnt-launch-on-unix
-sudo apt-get -y install \
-ca-certificates \
-fonts-liberation \
-libappindicator3-1 \
-libasound2 \
-libatk-bridge2.0-0 \
-libatk1.0-0 \
-libc6 \
-libcairo2 \
-libcups2 \
-libdbus-1-3 \
-libexpat1 \
-libfontconfig1 \
-libgbm1 \
-libgcc1 \
-libglib2.0-0 \
-libgtk-3-0 \
-libnspr4 \
-libnss3 \
-libpango-1.0-0 \
-libpangocairo-1.0-0 \
-libstdc++6 \
-libx11-6 \
-libx11-xcb1 \
-libxcb1 \
-libxcomposite1 \
-libxcursor1 \
-libxdamage1 \
-libxext6 \
-libxfixes3 \
-libxi6 \
-libxrandr2 \
-libxrender1 \
-libxss1 \
-libxtst6 \
-lsb-release \
-wget \
-xdg-utils 
+sudo apt-get -yqq install \
+  ca-certificates \
+  fonts-liberation \
+  libappindicator3-1 \
+  libasound2 \
+  libatk-bridge2.0-0 \
+  libatk1.0-0 \
+  libc6 \
+  libcairo2 \
+  libcups2 \
+  libdbus-1-3 \
+  libexpat1 \
+  libfontconfig1 \
+  libgbm1 \
+  libgcc1 \
+  libglib2.0-0 \
```

Comment:
@rohan-bes And also the spaces in front of each dependency.

Reply:
Ahh whitespace issues, I love em 😂 😭